ppcomp/ppcomp.py

```diff
@@ -373,47 +373,41 @@ def extract_footnotes_pp(self):
         """Extract footnotes from a PP text file. Text is iterable. Returns the text as an iterable,
         without the footnotes, and footnotes as a list of (footnote string id, line number of the
         start of the footnote, list of strings comprising the footnote). fn_regexes is a list of
-        (regex, fn_type) that identify the beginning and end of a footnote. The fn_type is 1 when
-        a ] terminates it, or 2 when a new block terminates it.
+        (regex, fn_type) that identify the beginning and end of a footnote. The fn_type is 2 when
+        a ] terminates it, or 1 when a new block terminates it.

```

Comment:
Bearing in mind that I'm not a PPer, when would a new block terminate a footnote, and not a closing square bracket?

Reply:
Most of the time, actually. There are an annoying number of styles, but even the default Guiguts fixup results in:
`[1] This is a footnote.`

Reply:
Ohh! Thanks. I was thinking as a formatter, not as a PPer. Silly me!